bin/lilith/src/main.rs

```diff
@@ -521,16 +333,11 @@ async fn realmain(args: Args, ex: Arc<Executor<'static>>) -> Result<()> {
     signals_handler.wait_termination(signals_task).await?;
     info!(target: "lilith", "Caught termination signal, cleaning up and exiting...");
 
-    // Save in-memory hosts to tsv file
-    save_hosts(&expand_path(&args.hosts_file)?, &lilith.networks).await;
-
```

Comment:
This was removed? Should be done upon exit.

Reply:
It now gets done in the refinery, as with all other nodes https://github.com/darkrenaissance/darkfi/blob/2dbaf413a0ed0059e2d8433f231f175207fa9513/src/net/hosts/refinery.rs#L70

Reply:
Ah great